crates/rolldown/tests/esbuild/import_star/re_export_star_external_es6/artifacts.snap

```diff
@@ -10,6 +10,8 @@ input_file: crates/rolldown/tests/esbuild/import_star/re_export_star_external_es
 ```js
 import "foo";
 
+export * from "foo"
+
 
 //#region entry.js
 var entry_ns = {};
```

Comment:
Was this more like a workaround or a correct fix? the `foo` is exported twice?

Reply:
Yeah. The current code rendering imports is kind of over abstraction. Don't want to deal with it in this timing.